thunder/tests/opinfos.py

```diff
@@ -1818,6 +1818,16 @@ def elementwise_binary_prims_generator(op, device, dtype, requires_grad, **kwarg
 
     yield SampleInput(a, b)
 
+    a = make_tensor((4, 4), device=device, dtype=dtype, requires_grad=requires_grad, **kwargs)
```

Comment:
Would you add a comment explaining what's being tested in each of these new cases?